Source/WebKit/UIProcess/API/Cocoa/_WKActivatedElementInfoInternal.h

```diff
@@ -37,14 +37,15 @@ namespace WebKit {
 @interface _WKActivatedElementInfo ()
 
 #if PLATFORM(IOS_FAMILY)
-+ (instancetype)activatedElementInfoWithInteractionInformationAtPosition:(const WebKit::InteractionInformationAtPosition&)information userInfo:(NSDictionary *)userInfo;
-- (instancetype)_initWithInteractionInformationAtPosition:(const WebKit::InteractionInformationAtPosition&)information userInfo:(NSDictionary *)userInfo;
++ (instancetype)activatedElementInfoWithInteractionInformationAtPosition:(const WebKit::InteractionInformationAtPosition&)information isUsingAlternateURLForImage:(BOOL)isUsingAlternateURLForImage userInfo:(NSDictionary *)userInfo;
```

Comment:
It looks like this method is always called with `isUsingAlternateURLForImage:NO`, so this method signature doesn't need to be changed.